src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationNewWithForceStrictCondition.java

```diff
@@ -17,11 +17,50 @@
 import java.io.InputStreamReader; //indent:0 exp:0
 
 public class InputIndentationNewWithForceStrictCondition { //indent:0 exp:0
-    void test() throws IOException  { //indent:4 exp:4
+    private java.util.List < ? > [ //indent:4 exp:4
+           ] arrayOfLists; //indent:11 exp:12 warn
+    public Object[] //indent:4 exp:4
+            variable1; //indent:12 exp:12
+    int variable2 //indent:4 exp:4
+    []; //indent:4 exp:12 warn
+    int[] //indent:4 exp:4
+    		variable3; //indent:12 exp:12
+
+	void test() throws IOException  { //indent:4 exp:4
         BufferedReader bf =  //indent:8 exp:8
                 new BufferedReader(  //indent:16 exp:16
                 new InputStreamReader(System.in) {  //indent:16 exp:24 warn
                     int a = 0; //indent:20 exp:28,32,36 warn
                 }); //indent:16 exp:24,28,32 warn
+
+        String[] tmp1 = new String[42 //indent:8 exp:8
+                                   + bf.toString().length()]; //indent:35 exp:16 warn
```

Comment:
please add to each statement/block with "warn", same by content but without warn.
To show  that here is code with correct indentation (no warn) and here is "same"  code but with violation.

Reply:
Added statements without warn